src/lib/apiTypesV2/BatchQuery.cpp

```diff
@@ -53,38 +53,6 @@ BatchQuery::~BatchQuery()
 
 /* ****************************************************************************
 *
-* BatchQuery::check - 
-*/
-std::string BatchQuery::check(ConnectionInfo* ciP, RequestType requestType)
```

Comment:
Yeah, I thought about  this a little bit also ...
In V2 parsing all requests that need to parse i.e. an attribute will use one unique function for that, same same with metadatas etc. And, all checks are done within that function.

Conclusion, no need for the check() methods in V2 parsing. 
So, those check methods that are not used by V1 parsing **should be removed**, just as is in this PR.

NTC